### DISCUSSION TITLE
Is it possible to support configuring automatic camel case conversion to format incoming and outgoing parameters?

### DISCUSSION THREAD

**Discussion by YaoJusheng** [apiflask/apiflask#623] [Q&A]
Sometimes the front-end and back-end code styles are not completely unified. 

I hope that the parameters passed in by the front-end will be in camel case, and then automatically converted to underscore form in the API, and then uniformly converted to camel case when returned to the front end.

Currently, I can only use `pre_load` and `post_dump` custom Schema as the parent class of other real Schemas to barely achieve this function.

But it cannot be automatically displayed correctly in docs documents, so is there a more elegant implementation of apiflask? 

Has anyone implemented this scenario?

**Comment by greyli** [comment]
Hello, it's a good advice. I have created https://github.com/apiflask/apiflask/issues/624 to track this feature. Closing this discussion now.
